client/src/app/services/profile.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService {
+
+  constructor() { }
+
+  publications: Array<any> = [];
+
+  getPublications() {
+    this.publications = [{
+      imageURL: "https://c.files.bbci.co.uk/48DD/production/_107435681_perro1.jpg",
+      title: "HeadlinePublications",
+      state: "activo",
```

Comment:
We currently don't have a "state" field in the Post interface, feel free to add it!